pkg/apis/componentconfig/edgecore/v1alpha1/types.go

```diff
@@ -471,3 +474,22 @@ type EdgeStream struct {
 	// default 15
 	WriteDeadline int32 `json:"writeDeadline,omitempty"`
 }
+
+// EdgeCluster indicates the edge cluster config
+type EdgeCluster struct {
+	// Enable indicates whether edge cluster is enabled, if set to false, skip checking other configs.
+	// default true
+	Enable bool `json:"enable"`
+
+	// clusterKubeconfig indicates the path to the edge cluster kubeconfig file
+	ClusterKubeconfig string `json:"clusterKubeconfig,omitempty"`
+
+	// Distribution of the cluster, supported value: arkots, to support in the furture: k3s, 
+	KubeDistribution string `json:"kubeDistribution,omitempty"`
+
+	// Name of the edge cluster
+	ClusterName string `json:"clusterName,omitempty"`
```

Comment:
Consider remove "Cluster" in the name. same with ClusterLabels as both are already in EdgeCluster struct

Reply:
Yep, I made the change.